### PR TITLE
Add Laravel 10 support

### PR DESCRIPTION
**Pull request by jyrkidn**

### CODE REVIEW

**Comment by jyrkidn** [comment]
I don't see immediately why the tests keep failing, I've added PHP 8.0 and Laravel 10 to the [exclude](https://github.com/ryangjchandler/blade-capture-directive/pull/12/files#diff-7314d0ebbd2e9537ae4889316745b4fd2fa43cb86275c9caae18a86ba228b642R26) section.

**Comment by ryangjchandler** [comment]
@jyrkidn I'll take a look in a little while.

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -13,14 +13,19 @@ jobs:
...
         include:
+          - laravel: 10.*
+            testbench: 8.*
           - laravel: 9.*
             testbench: 7.*
           - laravel: 8.*
             testbench: ^6.23
+        exclude:
+          - php: 8.0
+            laravel: 10
```
Comment (ryangjchandler) [inline .github/workflows/run-tests.yml]:
```suggestion
            laravel: 10.*
```

**Comment by owenvoke** [comment]
Looks like the tests are failing because L9 `--prefer-lowest` is installing Testbench `v7.0.0`, which uses Carbon 2.53.1, it needs to be Testbench `v7.9.0` or later (preferably `^7.22`) for this to not fail. 👀

**Pull request merged**